src/node_crypto.cc

```diff
@@ -813,7 +813,7 @@ void SecureContext::SetDHParam(const FunctionCallbackInfo<Value>& args) {
     return env->ThrowError("DH parameter is less than 1024 bits");
   } else if (size < 2048) {
     args.GetReturnValue().Set(FIXED_ONE_BYTE_STRING(
-        env->isolate(), "WARNING: DH parameter is less than 2048 bits"));
+        env->isolate(), "(node) WARNING: DH parameter is less than 2048 bits"));
```

Comment:
@silverwind this is logged in https://github.com/nodejs/node/blob/0803962860ae4cd2aa7355cbcfff3432e430e496/lib/_tls_common.js#L105, which is already handled in js code :)